src/status_im/ui/screens/chat/stickers/styles.cljs

```diff
@@ -10,4 +11,10 @@
    :width             icon-size
    :border-radius     (/ icon-size 2)
    :align-items       :center
-   :justify-content   :center})
\ No newline at end of file
+   :justify-content   :center})
+
+(defn stickers-panel-height []
+  (cond
+    platform/iphone-x? 299
```

Comment:
No way to retrieve this somehow?

Reply:
unfortunately no, we decided to hardcode these values